android/src/main/kotlin/io/didomi/fluttersdk/DidomiPlugin.kt

```diff
@@ -294,7 +297,7 @@ class DidomiPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
      */
     private fun getJavaScriptForWebView(result: Result) {
         try {
-            val text = Didomi.getInstance().javaScriptForWebView
+            val text = Didomi.getInstance().getJavaScriptForWebView()
```

Comment:
Why is this change necessary?

Reply:
During Didomi class migration to Kotlin, we merged `javaScriptForWebView` getter and `getJavaScriptForWebView(extra:String)` to `getJavaScriptForWebView(extra: String?)`